### ISSUE TITLE
Atom preview crashes when using & for search in url

### ISSUE THREAD

**Issue by mimamuh** [mjmlio/atom-mjml-preview#7]
Hi, atom preview crashes when I use a url with an & in its search params:
```jsx
href="http://mydomain.de/?email=myemail@mydomain.de&name=michael"
````

The error output in the atom console is:
```
/Users/mybook/.atom/packages/mjml-preview/node_modules/mjml-core/lib/Error.js:26
Uncaught (in promise) Error: [MJMLError] ParseError: Error while parsing the file(…)

at MJMLError.Error (native)
  at new MJMLError (/Users/mybook/.atom/packages/mjml-preview/node_modules/mjml-core/lib/Error.js:26:110)
  at documentParser (/Users/mybook/.atom/packages/mjml-preview/node_modules/mjml-core/lib/parsers/document.js:145:11)
  at MJMLRenderer.parseDocument (/Users/mybook/.atom/packages/mjml-preview/node_modules/mjml-core/lib/MJMLRenderer.js:137:45)
  at new MJMLRenderer (/Users/mybook/.atom/packages/mjml-preview/node_modules/mjml-core/lib/MJMLRenderer.js:125:12)
  at mjml2html (/Users/mybook/.atom/packages/mjml-preview/node_modules/mjml-core/lib/index.js:66:10)
  at MJMLView.renderMJML (/Users/mybook/.atom/packages/mjml-preview/lib/mjml-view.js:79:44)
  at MJMLView.render (/Users/mybook/.atom/packages/mjml-preview/lib/mjml-view.js:94:12)
  at /Users/mybook/.atom/packages/mjml-preview/lib/mjml-preview.js:74:27
```

**Comment by iRyusa** [comment]
Hi @MiMaMuh 

Thanks for this issue, it should be fixed with MJML 3.3 ( and should be released soon ) !

**Comment by ngarnier** [comment]
This was fixed with the release of [MJML v3.3.0](https://github.com/mjmlio/mjml/releases/tag/3.3.0) and [mjml-preview v1.3.0](https://github.com/mjmlio/atom-mjml-preview/releases/tag/v1.3.0) so I'm closing.
